regtech_mail_api/api.py

```diff
@@ -62,17 +62,20 @@ def read_root(request: Request):
 @app.post("/send")
 @requires("authenticated")
 async def send_email(request: Request):
-    headers = request.headers
-    subject = headers["X-Mail-Subject"]
     sender_addr = request.user.email
     sender_name = request.user.name
+    type = request.headers["case-type"]
+    subject = f"[DEV BETA] SBL Portal User Request for {type}" + (
+        f" by {sender_name}" if sender_name else ""
+    )
```

Comment:
Should we just drop the **_SBL Portal User Request for_** bit of the subject? By adding `type` we've already diverged from the proposed template anyway, right? I think `type` has all the info we need, and _SBL Portal_ isn't a term we use. For the demo, just seems like it'd be cleaner.

@billhimmelsbach, since you'll be demoing this, thoughts?

Reply:
I think we can definitely get rid of the `Portal`. Guess we can change the subject to anything we want since we don't want them to parse it and they're not parsing it now.

Also I'm introducing it, but I'd like @shindigira Is going to demo it.

Reply:
Per discussion, let's just drop the work **Portal** and move on. :)